packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -240,7 +242,8 @@ function TokenBalance({
 
     return sanitizeTokenSymbol(forToken.symbol, {
       erc20L1Address: forToken.address,
-      chain: on === NetworkType.l1 ? l1.network : l2.network
+      chain: on === NetworkType.l1 ? l1.network : l2.network,
+      skipUSDCoverride
```

Comment:
If I understand correctly, this is only set to true, for L2 TokeBalance no? Can we use that, so we don't need to pass it as an argument

Reply:
There are two `<TokenBalance/>` for USDC
but for others they use the normal one without this arg